tests/testing/LocalBearTestHelperTest.py

```diff
@@ -49,6 +49,11 @@
         ]),
     settings={'max_line_length': 20},
 )
+ChangeBaseDirectoryTest = verify_local_bear(TestBear,
```

Comment:
This test doesn't seem thorough. You're supposed to test if the directory is actually changing when you supply base_directory so I'd suggest mocking `change_directory` and testing if it is called with the right arguments and the right number of times.

I recommend consulting one other reviewer though.

Reply:
Yeah, this doesn't test the working of the changes made. Since even if base_directory isin't provided the test would pass.

Reply:
Yeah Agreeing with both the reviewers above you could try mocking `change_directory`, or if you are not familiar with mocking you could follow the assert,check and test method and see if you are able to solve the issue.